### ISSUE TITLE
Doesn't seem to work when loaded from within an iFrame

### ISSUE THREAD

**Issue by softforge** [hampusohlsson/browser-deeplink#5]
I have tried to implement this on a page which is loaded within an iFrame but it does not seem to work on my Android (Chrome v40). Is there a reason why it wouldn't work in an iFrame?

**Comment by hampusohlsson** [comment]
Interesting use case. There is another issue is about iframes not working on Chrome/Android, so it might just be the case it does not work regardless of the nested iframe.

**Comment by softforge** [comment]
Ah yes, I see, you might be right. I'll do some more testing to see if it is actually the other issue I am getting.

Thanks, I would imagine that the Android emulator exhibits the same behaviour. It comes with Android Studio which is easy to setup if you want to try and test the issue.

**Comment by godspeedelbow** [comment]
It doesn't work the first time (page url gets set to `intent://...`). When I press back, it does work. This is very consistent

**Comment by mrozlukasz** [comment]
Yup same behaviour like @godspeedelbow. No deeplink with my app's uri but intent://. The fallback triggers only if I press back button twice.
